docs/sources/installation/fedora.md

```diff
@@ -6,111 +6,192 @@ page_keywords: Docker, Docker documentation, Fedora, requirements, linux
 
 Docker is supported on the following versions of Fedora:
 
-- [*Fedora 20 (64-bit)*](#fedora-20-installation)
-- [*Fedora 21 and later (64-bit)*](#fedora-21-and-later-installation)
+- Fedora 20 
+- Fedora 21 and later 
 
-Currently the Fedora project will only support Docker when running on kernels
-shipped by the distribution. There are kernel changes which will cause issues
-if one decides to step outside that box and run non-distribution kernel packages.
+This page instructs you to install using Docker-managed release packages and
+installation mechanisms. Using these packages ensures you get the latest release
+of Docker. If you wish to install using Fedora-managed packages, consult your
+Fedora release documentation for information on Fedora's Docker support.
+
+##Prerequisites
 
-## Fedora 21 and later
+Docker requires a 64-bit installation regardless of your Fedora version. Docker
+requires that your kernel must be 3.10 at minimum.  To check your current kernel
+version, open a terminal and use `uname -r` to display your kernel version:
 
-### Installation
+    $ uname -r 
+    3.19.5-100.fc20.x86_64
 
-Install the Docker package which will install Docker on our host.
+If your kernel is at a older version, you must update it.
```

Comment:
We always advise people to make sure their system is fully up-to-date. Something to consider here as well (just a mention)?